### ISSUE TITLE
true color support ?

### ISSUE THREAD

**Issue by mckellygit** [tomscii/zutty#105]
Hi,
Thank you for zutty.
For me it seems true color support is not there, but perhaps I am not using a correct/up-to-date xterm-256color terminfo db or some other config option ?  Is there a test you suggest I use to verify true color support ?
From some printf esc codes for rgb values and from some vim highlight settings and some other programs I use it seems to me I do not have true colors working properly.
It appears 256 colors work fine in zutty, and true colors work in other terminals.
thx,
-m

**Comment by tomscii** [comment]
https://github.com/tomscii/zutty/blob/master/test/truecolor_inc.sh

**Comment by mckellygit** [comment]
ok, thank you.
I did almost the same test, now I understand more, outside of tmux, I agree this looks good and matches other terminals.  Inside of tmux its 256 colors, so I add terminal-overrides 'xterm*:Tc' and now I think the true color test is good, but vim highlight attributes reverse and bold for some combinations are not correct.  For example:
:hi DiffAdd  cterm=reverse ctermfg=142 ctermbg=235
is no longer correct when I have Tc or RGB enabled in tmux.
I will research it more.
-m

**Comment by aptudock** [comment]
I ran into the same issue. Even though zutty supports truecolor, applications might not be able to detect it.
Zutty could help with that by announcing this capability via setting the env variable COLORTERM=truecolor.

**Comment by tomscii** [comment]
> I ran into the same issue. Even though zutty supports truecolor, applications might not be able to detect it. Zutty could help with that by announcing this capability via setting the env variable COLORTERM=truecolor.

Thanks, good suggestion. I have added this in the latest commit.

Regarding tmux, I found this helpful: https://github.com/tmux/tmux/wiki/FAQ#how-do-i-use-rgb-colour
Specifically, adding the line `set-option -sa terminal-overrides ",xterm*:Tc"` to my `.tmux.conf` solved the issue.
Then I had to close and re-create my session (just reloading the config file and detaching/reattaching was *not* enough).
This with tmux 3.1c
